Source/SlimDetours/Memory.c

```diff
@@ -130,6 +130,16 @@ detour_memory_free(
     return RtlFreeHeap(_detour_memory_heap, 0, BaseAddress);
 }
 
+VOID
+detour_memory_uninitialize(VOID)
+{
+    if (_detour_memory_heap != NULL)
+    {
+        RtlDestroyHeap(_detour_memory_heap);
+        _detour_memory_heap = NULL;

```

Comment:
```suggestion
        _detour_memory_heap = RtlDestroyHeap(_detour_memory_heap);
        return _detour_memory_heap != NULL;
```
We could returns a `BOOL` or `STATUS_INVALID_HANDLE` (`HeapDestroy` returns `ERROR_INVALID_HANDLE` if `RtlDestroyHeap` fails) indicating whether it was successful or not.

Reply:
Done.